README.md

```diff
@@ -34,21 +34,17 @@ export TEST_ARCHIVIST="https://app.rkvst.io"
 export TEST_AUTHTOKEN_FILENAME=credentials/.auth_token
 export TEST_NAMESPACE="unique label"
 export TEST_VERBOSE=-v
-export TEST_PROOF_MECHANISM="--proof-mechanism=SIMPLE_HASH"
 ```
 
 If TEST_VERBOSE is "-v" debugging output will appear when running the examples. Otherwise leave blank or undefined.
 
-TEST_PROOF_MECHANISM should be "KHIPU" or "SIMPLE_HASH". If unspecified the default is "SIMPLE_HASH"
-
 Windows using Powershell - at the command prompt set values for environment variables:
 
 ```bash
 $Env:TEST_ARCHIVIST="https://app.rkvst.io"
 $Env:TEST_AUTHTOKEN_FILENAME = '<path of token location>'
 $Env:TEST_NAMESPACE = Get-Date -UFormat %s
 $Env:TEST_VERBOSE = '-v'
-$Env:TEST_PROOF_MECHANISM="--proof-mechanism=SIMPLE_HASH"
```

Comment:
The problem with removing altogether is that in about 2 months we'll have the Khipu replacement and will have to put it all back again. Especially given that testing the new stuff is important I think we should leave this